stubs/jsonschema/jsonschema/_format.pyi

```diff
@@ -1,35 +1,42 @@
-from typing import Any
+from typing import Any, Iterable
 
 class FormatChecker:
     checkers: Any
-    def __init__(self, formats: Any | None = ...) -> None: ...
+    def __init__(self, formats: Iterable | None = ...) -> None: ...
```

Comment:
```suggestion
    def __init__(self, formats: Iterable[str] | None = ...) -> None: ...
```

Reply:
Sorry, I pushed too soon. Thanks for the help!